interface/patient_file/transaction/add_transaction.php

```diff
@@ -317,21 +308,25 @@ function submitme() {
 <form name='new_transaction' method='post' action='add_transaction.php?transid=<?php echo htmlspecialchars( $transid, ENT_QUOTES); ?>' onsubmit='return validate(this)'>
 <input type='hidden' name='mode' value='add'>
 
-	<table>
-	    <tr>
-            <td>
-                <b><?php echo htmlspecialchars( xl('Add/Edit Patient Transaction'), ENT_NOQUOTES); ?></b>&nbsp;</td><td>
-                 <a href="javascript:;" class="css_button" onclick="submitme();">
-                    <span><?php echo htmlspecialchars( xl('Save'), ENT_NOQUOTES); ?></span>
-                 </a>
-             </td>
-             <td>
-                <a href="transactions.php" class="css_button" onclick="top.restoreSession()">
-                    <span><?php echo htmlspecialchars( xl('Cancel'), ENT_NOQUOTES); ?></span>
-                </a>
-            </td>
-        </tr>
-	</table>
+    <div class="container-fluid">
+        <div class="row">
+            <div class="col-xs-12">
+                <div class="page-header">
+                    <h1><?php echo xl('Add/Edit Patient Transaction');?></h1>
+                </div>
+            </div>
+            <div class="col-xs-12">
+                <div class="btn-group">
+                    <a href="#" class="btn btn-default btn-save" onclick="submitme();">
+                        <?php echo htmlspecialchars( xl('Save'), ENT_NOQUOTES); ?>
+                    </a>
+                    <a href="transactions.php" class="btn btn-link btn-cancel" onclick="top.restoreSession()">
+                        <?php echo htmlspecialchars( xl('Cancel'), ENT_NOQUOTES); ?>
```

Comment:
This cancel button/link seems a bit odd since it's so close to the below Transaction Type selector